src/util/deleteContentByMod.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { MessageType } from 'antd/es/message/interface';
+import { NotificationStatus } from '~src/types';
+import queueNotification from '~src/ui-components/QueueNotification';
+import nextApiClientFetch from './nextApiClientFetch';
+import { ProposalType } from '~src/global/proposalType';
+
+export async function deleteContentByMod(postId:number|string|undefined, proposalType:ProposalType, reason:string, commentId?:string, replyId?:string, onDeleteComment?:()=>void , onDeleteReply?:()=>void, onDeletePost?:()=>void)   {
```

Comment:
postId shouldn't be undefined

Reply:
`onDeleteComment?:()=>void` , `onDeleteReply?:()=>void`, `onDeletePost?:()=>void` can be changed to a single `onSuccess()`

Reply:
Resolved, pls check